app/controllers/restaurants_controller.rb

```diff
@@ -34,7 +34,12 @@ def update
     redirect_to restaurant_path(@restaurant)
   end
 
-  def delete
+  def destroy
+    @restaurant = Restaurant.find(params[:id])
+    @restaurant.destroy
+    if @restaurant.destroy
+      redirect_to restaurants_path, notice: "Restaurant deleted"
+    end
```

Comment:
You need an else here to do something in case the `.destroy` doesn't happen (should show an error message)